cssselect/xpath.py

```diff
@@ -332,6 +343,9 @@ def xpath_element(self, selector):
         if not element:
             element = '*'
             safe = True
+        if element == '<':
+            element = '<'
```

Comment:
why the reassignment to `<` ?

Reply:
Previously was reassigning to '.', will fix :)

Reply:
Fixed.